package.json

```diff
@@ -31,8 +31,8 @@
   "scripts": {
     "postinstall": "cd src/app && npm install --no-optional",
     "start": "npm run start:dev",
-    "start:dev": "electron . --update-url=DISABLED --cache-directory=./src/web --disable-http-cache",
-    "start:build": "npm run build:web && electron . --update-url=DISABLED --cache-directory=./build/latest",
+    "start:dev": "electron . --update-url=DISABLED --cache-directory=./src/web --disable-http-cache --remote-debugging-port=9200",
```

Comment:
Why setting/changing the remote debugging port?

Reply:
This is the only way I was able to connect the deuger in webstorm
But this is my first contact with an electron
Should I roll back this change?

Reply:
Yes, leave the remote debugging off (it could have side-effects and a website may be able to detect the browser is in automation mode => bot detection).

You can use the chrome debugger by pressing F12 in HakuNeko, or if really want remote debugging, use the command `npm run start:dev -- --remote-debugging-port=9200` to start HakuNeko from your IDE.